.omeroci/app-config

```diff
@@ -0,0 +1,6 @@
+set -e
+set -u
+OMERO_VENV=${OMERO_VENT:-/opt/omero/web/venv}
```

Comment:
typo in `OMERO_VENT`

Reply:
Discussed today with @joshmoore we are going to remove that file by default and configure the app directly test-infra